include/swift/AST/Decl.h

```diff
@@ -5008,6 +5008,9 @@ class FuncDecl final : public AbstractFunctionDecl,
   }
 
   bool isGetter() const { return getAccessorKind() == AccessorKind::IsGetter; }
+  bool isComputedGetter() const {
```

Comment:
Ah, sorry, this isn't correct. "Computed" just means there's a getter implementation at all (as opposed to "stored"). And again, checking the location of the braces is incorrect.